src/components/comment-adder.jsx

```diff
@@ -0,0 +1,63 @@
+import { useState } from "react";
+import { postComment } from "../utils/api-calls";
+import "../styles/comment-adder.css";
+
+export default function CommentAdder({ article_id, setComments, user }) {
```

Comment:
If your not logged in and attempt to comment, the comment isnt added but you do not get some feedback informing you why.